3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,104 @@
+ //hello... Hello... This is my first experience in Javascript :)
+
+console.log("******    Age-ify (A future age calculator)    ******");
+let myYearOfBirth = 1991;
+let yearFuture = 2027;
+let age = yearFuture - myYearOfBirth;
+
+console.log("I\'ll be " + age + " years old in " + yearFuture + ".");
+console.log("\n");
+
+
+
+console.log("******    Goodboy-Oldboy (A dog age calculator)    ******");
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2045;
+// We have a conditional statement, so a "BOOLEAN variable" can represent (shouldShowResultInDogYears).
+let shouldShowResultInDogYears = true;
+
+ // According to the information in the txt:(1 human year = 7 dog year)
+let dogAge = (dogYearFuture - dogYearOfBirth)*7;
+let humanAge = dogYearFuture - dogYearOfBirth;
+
+if (shouldShowResultInDogYears) {
+    console.log("His dog will be " + dogAge + " dog years old in " + dogYearFuture + ".");
+}
+
+else {
+ console.log("His dog will be " + humanAge + " human years old in " + dogYearFuture + ".");
+}
+console.log("/* His dog will be 28 human years old in 2045. */");
+console.log("\n");
+
+
+console.log("******    Housey pricey (A house price estimator)    ******");
+//Peter:
+var petersPreferredHouseWide = a;
```

Comment:
- `var`  keyword is not really used in javascript anymore, we use only `const` or `let` these days, please replace the `var` keywords with `const` 

- this is not an issue anymore but since the Javascript is synchronous language (its read from top to bottom) 
we should always first declare a value and then use it as this 
```
var a = 8; //m
var b = 10; //m
var c = 10; //m

const petersPreferredHouseWide = a;
const petersPreferredHouseDeep = b;
const petersPreferredHouseHigh = c;
```
Also, I think using a,b,c variables is a bit redundant here. you can simply assign the value directly to the variables as

```
const petersPreferredHouseWide = 8;
const petersPreferredHouseDeep = 10;
const petersPreferredHouseHigh = 10;
```